test/another-command/build.gradle

```diff
@@ -0,0 +1,6 @@
+apply plugin: "de.flachnetz.golang-gradle-plugin"
+
+golang {
+  binaryName = "another-command"
+  cgoEnabled = true
+}
```

Comment:
for sub projects we want to build we need to just apply the plugin and add a binaryName.